sentinel/transitions/update_clinics.js

```diff
@@ -71,6 +72,24 @@ var getContact = function(db, doc, callback) {
     }
 };
 
+var getHydratedContact = function(db, doc, callback) {
+    getContact(db, doc, function(err, contact) {
```

Comment:
There's a small performance improvement we can make here. Because we're hydrating the contact in line 83 we don't need to get the contact here, just the contact id. Change this function to `getContactId` and remove the `include_docs: true` options from the function, and return just the id.